### DISCUSSION TITLE
OpenAI and Whisper country availability

### DISCUSSION THREAD

**Discussion by erictice** [openai/whisper#653] [Q&A] [answered]
I have read some posts suggesting that OpenAI is not available for use in Saudi Arabia and a couple of other countries.  As many of the components like Whisper are open source and presumably not SaaS in nature what is the potential limitation here?  Or is there one?  The assumption is that I can take Whisper and run it anywhere as I have the source code here.   Please provide any clarification on this or required other components that would limit its usage in SA or other countries.

**Comment by jongwook** [comment, marked as answer]
Passing along what our lawyer said:

> There’s no limitation on anything we open source besides whatever is in the license.
